openmrs/src/test/java/org/motechproject/openmrs/it/MRSTasksIntegrationBundleIT.java

```diff
@@ -235,6 +235,26 @@ public void testOpenMRSProviderDataSourceAndCreateEncounterAction() throws Inter
     }
 
     @Test
+    public void testCreateEncounterPostActionParameter() throws InterruptedException {
+        Task task = prepareCreateEncounterPostActionParameterTask();
+
+        getTriggerHandler().registerHandlerFor(task.getTrigger().getEffectiveListenerSubject());
+
+        activateTrigger();
+
+        // Give Tasks some time to process
+        assertTrue(waitForTaskExecution(task.getId()));
+
+        Encounter encounter = encounterService.getLatestEncounterByPatientMotechId(DEFAULT_CONFIG_NAME, "654", createdEncounterType.getName());
```

Comment:
as far as I understand "654" is a motechId of createdPatient object, so it will be more readable to pass this parameter using getter